### PR TITLE
Document how to find tool IDs

### PR DESCRIPTION
**Pull request by hexylena**
thanks @annefou for pointing out it was unclear!

### CODE REVIEW

File: topics/contributing/tutorials/create-new-tutorial-content/tutorial.md [hunk 1]
```diff
@@ -510,6 +510,20 @@ When viewed through Galaxy, students will see:
 
 <span data-tool="upload1" title="Tested with upload1" class="tool galaxy-proxy-active"><strong>Import some data</strong> <i class="fas fa-wrench" aria-hidden="true"></i><i aria-hidden="true" class="fas fa-cog"></i><span class="visually-hidden">Tool: upload1</span></span>
 
+### How to find these IDs?
+
+The first and easiest way is to simply use planemo to generate the training from a workflow. In recent versions of planemo, this is managed automatically.
```
Comment (nsoranzo) [inline topics/contributing/tutorials/create-new-tutorial-content/tutorial.md:515]:
```suggestion
The easiest way is to use planemo to generate the training from a workflow. In recent versions of planemo, this is managed automatically.
```

File: topics/contributing/tutorials/create-new-tutorial-content/tutorial.md [hunk 2]
```diff
@@ -510,6 +510,20 @@ When viewed through Galaxy, students will see:
...
+
+The alternative is to figure out the ID for the tool you want to use:
+
+1. Find your tool in galaxy, and click to access the tool form.
```
Comment (nsoranzo) [inline topics/contributing/tutorials/create-new-tutorial-content/tutorial.md:519]:
```suggestion
1. Find your tool in Galaxy, and click to access the tool form.
```

File: topics/contributing/tutorials/create-new-tutorial-content/tutorial.md [hunk 3]
```diff
@@ -510,6 +510,20 @@ When viewed through Galaxy, students will see:
...
+2. Click on Options at the top right
+3. Click on Share
+4. The URL has the tool ID. e.g. `https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/galaxyp/mz_to_sqlite/mz_to_sqlite/2.0.4+galaxy1`
```
Comment (nsoranzo) [inline topics/contributing/tutorials/create-new-tutorial-content/tutorial.md:522]:
```suggestion
4. The URL shown will be something like `https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/galaxyp/mz_to_sqlite/mz_to_sqlite/2.0.4+galaxy1`
```

File: topics/contributing/tutorials/create-new-tutorial-content/tutorial.md [hunk 4]
```diff
@@ -510,6 +510,20 @@ When viewed through Galaxy, students will see:
...
+5. Keep only the part after the `=`, so `toolshed.g2.bx.psu.edu/repos/galaxyp/mz_to_sqlite/mz_to_sqlite/2.0.4+galaxy1`
```
Comment (nsoranzo) [inline topics/contributing/tutorials/create-new-tutorial-content/tutorial.md:523]:
```suggestion
5. Keep only the part after the `=`, so `toolshed.g2.bx.psu.edu/repos/galaxyp/mz_to_sqlite/mz_to_sqlite/2.0.4+galaxy1` in this example
```

**Comment by annefou** [comment]
Super! Thank you!

**Comment by bgruening** [comment]
Merci!

**Pull request merged**